src/gadget_user.erl

```diff
@@ -0,0 +1,26 @@
+-module(gadget_user).
+
+-author('elbrujohalcon@inaka.net').
+
+-behaviour(application).
+
+-export([ start/0
+        , stop/0
+        , start/2
+        , stop/1
+        ]).
+
+-spec start() -> {ok, _} | {error, term()}.
+start() -> application:ensure_all_started(gadget_user).
+
+-spec stop() -> ok | {error, term()}.
+stop() -> application:stop(gadget_user).
+
+-spec start(atom(), any()) -> {ok, pid()} | {error, term()}.
+start(normal, _Args) -> {ok, self()}.
+
+-spec stop(atom()) -> ok.
+stop(_State) -> ok.
+
+%something(Wrong) -> Here.
+%% LOOK MA, A VERY VERY VERY LONG LINE WITH ALL CAPS AND REALLY NO MEANING AT ALL
```

Comment:
Line 26 is too long: "%% LOOK MA, A VERY VERY VERY LONG LINE WITH ALL CAPS AND REALLY NO MEANING AT ALL".